src/utils/localisationHelpers.ts

```diff
@@ -17,9 +32,11 @@ const getCustomCalendarLocale = (
     if (!customCalendar) {
         return undefined
     }
+
     const customLocalisations = customCalendar.locales || {}
     const result =
-        (locale && customLocalisations?.[locale]) ??
+        customLocalisations?.[locale!] ??
```

Comment:
I'm a bit confused by the syntax change to assert `[locale!]` since the function definition says that locale can be `undefined`, but I asssume this is right 😊

Reply:
yeah it's a fair point .. I updated it to `locale && customLocalisations[locale]`